en/simulation/gazebo.md

```diff
@@ -242,11 +242,16 @@ PX4 SITL for Gazebo supports UDP video streaming from a Gazebo camera sensor att
 
 ### Prerequisites
 
-Install *Gstreamer 1.0* and its dependencies:
+Ubuntu: Install *Gstreamer 1.0* and its dependencies:
```

Comment:
Thanks. FWIW the setup scripts for Linux do this set automatically except for the `gstreamer1.0-hybris` dependency: https://github.com/PX4/Firmware/blob/master/Tools/setup/ubuntu.sh#L200

I think worth leaving this in though